caravel/templates/manage_listings.html

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.html" %}
+{% block title %} Manage Listings {% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block content %}
```

Comment:
Am not entirely clear what you mean here. 

My manage listings route is like: 

```
@app.route("/manage_listings", methods=['GET'])
def manage_listings():
    email = session.get("email")
    if not email:
        abort(403)
    listings = helpers.run_query(email=session.get("email"))
    return render_template("manage_listings.html", listings=listings)
```

Seems like I'm filtering listings by making a query. 
Do you mean not to separate keyword and email parts of the query?

Seems like both in function and code, what I have in the manage_listings route is the same as the index route.